qmlti/src/resources/PerceptionSoap.php

```diff
@@ -187,11 +187,10 @@ public function get_assessment($assessment_id) {
     return $response->Assessment;
   }
 
-  public function get_assessment_list($parent_id, $only_run_from_integration) {
+  public function get_assessment_list($parent_id) {
```

Comment:
Missing PHPDoc. Probably a good idea if someone else has to maintain this in the future.